tasks/main.yml

```diff
@@ -94,6 +94,12 @@
     state: "{{ item.state|default('present') }}"
   with_items: "{{ s_cd.groups }}"
 
+- name: Common | Refresh Host SSH
+  include_tasks: refresh_ssh_host_key.yml
```

Comment:
change this to "import_" instead of "include_" and just remove all the tags in importer file.